updated guidance for TFE disk requirements (#2170)
hashicorp/terraform-website: content/enterprise/system-overview/capacity.mdx

The amount of disk storage available to a system plays a small role in the capacity of an instance.
A root volume with 200 GB of storage can sustain a capacity well over 100 concurrent runs.

> 
> 

Reviewer: The extra space between the text and the heading isn't needed. It'll render the same as long as there's at least 1 empty line in between.

```suggestion
```

but in general we suggest 35 IOPS per concurrent Terraform run. So if an instance is configured for
20 concurrent runs, the disk should have 700 IOPS allocated. For reference, on AWS, an EBS volume
with an allocated size of 250 GB comes with a steady state of 750 IOPS.

> It is recommended to use a disk with a minimum of 500 IOPS or 100+MB/s throughput, whichever is the smaller and these are provisioned values.

Reviewer: Where does this recommendation come from?

Author: https://support.hashicorp.com/hc/en-us/articles/4409044739859-Container-ptfe-base-startup-failed

Author: Solution 2

Reviewer: I'm an engineer on the Terraform Enterprise team and I haven't seen any official recommendation for IOPS and/or throughput aside from the paragraph right above this.

> This resource is harder to predict than memory or CPU usage because it varies per Terraform module, but in general we suggest 35 IOPS per concurrent Terraform run. So if an instance is configured for 20 concurrent runs, the disk should have 700 IOPS allocated. For reference, on AWS, an EBS volume with an allocated size of 250 GB comes with a steady state of 750 IOPS.

In our load testing, we've generally used a disk type that favors I/O, such as `io2` (AWS) with 700 IOPS. 

However, to stay consistent with Support and since this is a minimum recommendation, I would recommend this wording.

```suggestion
It is recommended to use a disk with at least 500 IOPS or 100+ MB/s throughput, whichever is lower.
```
